tests/.netrc

```diff
@@ -0,0 +1,3 @@
+machine test_entrc.httpx.com
+login tomchristie
+password password123
```

Comment:
I'd probably marginally prefer that we use a fake style username here `example-username` and `example-password` or something. Just feels marginally cleaner.